articles/tutorials/index.md

```diff
@@ -0,0 +1,10 @@
+# Tutorials
+
+We try to help you understand Intent Architect as a newcomer by providing a couple of tutorials to work through.
+Please choose the one that closely relates to the feature you're trying to learn.
+
+### [Getting Started - Create an ASP.NET Core web app](create_an_aspnetcore_web_app.md)
```

Comment:
Let's change this to an xref with a blank titles so it will automatically populate the title from the article.